justfile

```
@@ -2,6 +2,9 @@
 # ^ A shebang isn't required, but allows a justfile to be executed
 #   like a script, with `./justfile test`, for example.
 
+# Windows users will need to enable this unless running under something like Git for Windows, GitHub Desktop, or Cygwin.
+#set windows-powershell := true
```

Let's just make it the default, I think it's probably better that way:

```suggestion
set windows-powershell := true
```